README.md

```diff
@@ -5,117 +5,93 @@
+There are three key concepts in Spectral: **Rulesets**, **Rules**, and **Functions**.
 
-- **Ruleset** is a container for a collection of rules and functions.
-- **Rule** filters your object down to a set of target values, and specify the function that should evaluate those values.
-- **Function** accept a value and return issue(s) if the value is incorrect.
+- **Rulesets** act as a container for rules and functions.
+- **Rules** filter your object down to a set of target values, and specify the function that is used to evaluate those values.
+- **Functions** accept a value and return issue(s) if the value is incorrect.
 
-Think of a set of **rules** and **functions** as a flexible and customizable style guide for your JSON objects.
+Think of a **Spectral ruleset** as a flexible and customizable style guide for your JSON objects.
 
 ## Programmatic usage
 
-Spectral is written in TypeScript (JavaScript) and can be used directly for when you need to use Spectral programmatically. Take a look at our ["JavaScript API documentation"](docs/js-api.md).
+Spectral is written in TypeScript (the compiled to JavaScript) and can be used directly for when you need to use Spectral programmatically. Take a look at our ["JavaScript API documentation"](docs/js-api.md).
 
 ## FAQs
 
-**How is this different than [Ajv](https://github.com/epoberezkin/ajv)?**
+#### How is this different than [Ajv](https://www.npmjs.com/package/ajv)?
 
 Ajv is a JSON Schema validator, not a linter. Spectral does expose a `schema` function that you can use in your rules to validate all or part of the target object with JSON Schema (Ajv is used under the hood). However, Spectral also provides a number of other functions and utilities that you can use to build up a linting ruleset to validates things that JSON Schema is not well suited for.
 
-**I want to lint my OpenAPI documents but don't want to implement Spectral right now.**
+#### I want to lint my OpenAPI documents but don't want to implement Spectral right now.
 
 No problem! A hosted version of Spectral comes **free** with the Stoplight platform. Sign up for a free account [here](https://stoplight.io/?utm_source=github&utm_campaign=spectral).
 
-**What is the difference between Spectral and [Speccy](https://github.com/wework/speccy)?**
+#### What is the difference between Spectral and [Speccy](https://github.com/wework/speccy)?
 
 With Spectral, lint rules can be applied to _any_ JSON object. Speccy is designed to work with OpenAPI v3 only. The rule structure is different between the two. Spectral uses [JSONPath](http://goessner.net/articles/JsonPath/) `path` parameters instead of the `object` parameters (which are OpenAPI specific). Rules are also more clearly defined (thanks to TypeScript typings) and now require specifying a `type` parameter. Some rule types have been enhanced to be a little more flexible along with being able to create your own rules based on the built-in and custom functions.
 
+### Executable binaries
```

Comment:
Why not include this with all the other installations?

Reply:
It's just a longer section that's less relevant to most folks (majority I assume will use NPM or Docker). I don't have a strong opinion on this though, we can move it back up if you want.

Reply:
This makes me wonder a little bit why we're offering binaries